packages/seq/seq.base/opam

```diff
@@ -0,0 +1,8 @@
+opam-version: "1.2"
+maintainer: " "
+authors: " "
+homepage: " "
+depends: ["ocamlfind" {build}]
```

Comment:
Out of curiosity - why does this require `ocamlfind`?

Reply:
Because that's the simplest way I know to install the `META` file at the proper place. I'm not on a crusade to kill ocamlfind.

Reply:
I was only asking because I don't see it used - does `opam-installer` use `ocamlfind` internally to install `META` from your `seq.install` file?